include/wil/wistd_functional.h

```diff
@@ -250,21 +250,24 @@ namespace wistd     // ("Windows Implementation" std)
             return _Invoker::__call(__f_, wistd::forward<_ArgTypes>(__arg)...);
         }
 
+        // 'wistd::function' is most similar to 'inplace_function' in that it _only_ permits holding function objects
+        // that can fit within its internal buffer. Therefore, we expand this size to accommodate space for at least 12
+        // pointers (__base vtable takes an additional one).
+        constexpr const size_t __buffer_size = 13 * sizeof(void*);
+
     }  // __function
 
+    // NOTE: The extra 'alignas' here is to work around the x86 compiler bug mentioned in
+    // https://github.com/microsoft/STL/issues/1533 to force alignment on the stack

```

Comment:
Without this, passing `wistd::function` by value as function arguments can cause alignment issues